src/components/Textarea/Textarea.tsx

```diff
@@ -32,7 +32,7 @@ function Textarea(props: InternalTextareaProps) {
     helpText,
     disabled,
     optional,
-    error,
+    error = [],
```

Comment:
Was there any issues before this change ?

Reply:
yes the `hasError` was not working

Reply:
I will update the code to remove the necessity of this